pkg/templates/mixer/TemplateExtensions.proto

```diff
@@ -0,0 +1,19 @@
+syntax = "proto3";
+
+package istio.mixer.v1.config.template;
+import "google/protobuf/descriptor.proto";
+
+enum TemplateVariety {
+    TEMPLATE_VARIETY_UNSPECIFIED = 0;
```

Comment:
Do we need/want an UNSPECIFIED variant? We have it in our APIs due to how proto3 handles default values, but I'm not sure the same reasoning applies to a proto extension.